### PR TITLE
LRQA-68220 Write Poshi Tests to Validate Hidden Field on Forms

### PR DESCRIPTION
**Pull request by leticiachiu**
Related Issue: https://issues.liferay.com/browse/LRQA-68220

### CODE REVIEW

**Comment by liferay-forms** [comment]
ci:test:sf

**Comment by liferay-forms** [comment]
Hi, you don’t need to run the CI commands anymore, I’ll do it for you and let you know if any issue happens.

**Comment by liferay-forms** [comment]
ci:test:relevant

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase [hunk 1]
```diff
@@ -692,6 +692,114 @@ definition {
...
+
+		FormFields.enableSwitch(fieldName = "Hide Field");
+
+		Form.viewHiddenField(fieldLabel = "Numeric");
+
+		for (var switchName : list "Show Label,Repeatable,Require Confirmation,Validation,Input Mask") {
+			AssertElementPresent(
+				key_switchOption = "${switchName}",
+				locator1 = "FormFields#SWITCH_NOT_VISIBLE");
+		}
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsNumericField.testcase:717]: Great job, @leticiachiu! :clap:

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase [hunk 2]
```diff
@@ -1089,6 +1089,58 @@ definition {
...
+
+		for (var switchName : list "Show Label,Repeatable,Require Confirmation,Validation") {
+			AssertElementPresent(
+				key_switchOption = "${switchName}",
+				locator1 = "FormFields#SWITCH_NOT_VISIBLE");
+		}
+
+		AssertElementPresent(
+			key_switchOption = "Autocomplete",
+			locator1 = "Form#SIDEBAR_NOT_VISIBLE_TAB");
```
Comment (natocesarrego) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase:1115]: Nice catch and job, @leticiachiu! :tada: :smile:

**Review by natocesarrego — APPROVED** [review]
Lovely pull, @leticiachiu!

**Comment by natocesarrego** [comment]
ci:forward

**Pull request closed without merging**